data_pipeline_api/file_formats/parameter_file.py

```diff
@@ -0,0 +1,109 @@
+import toml
+from io import TextIOBase
+from enum import Enum
+from numbers import Real
+from typing import Union, Sequence, Dict, Any
+import numpy as np
+from scipy import stats
+
+# ======================================================================================
+# Common
+# ======================================================================================
+
+ParameterComponent = Dict[str, Any]
+Estimate = Real
+Distribution = Union[stats.rv_discrete, stats.rv_continuous]
+Samples = np.ndarray
+
+
+class Type(Enum):
+    point_estimate = "point-estimate"
```

Comment:
Aren't those usually all capitals to indicate a constant?

Reply:
yep ok, I do think it looks a bit grim but that is the convention, I'll change it